chirp/drivers/mml_jc8810.py

```diff
@@ -0,0 +1,1070 @@
+            mem.name += str(char)
+        mem.name = mem.name.rstrip()
+
+        dtcs_pol = ["N", "N"]
+
+        if _mem.txtone in [0, 0xFFFF]:
+            txmode = ""
+        elif _mem.txtone >= 0x0258:
+            txmode = "Tone"
+            mem.rtone = int(_mem.txtone) / 10.0
+        elif _mem.txtone <= 0x0258:
+            txmode = "DTCS"
+            if _mem.txtone > 0x69:
+                index = _mem.txtone - 0x6A
+                dtcs_pol[0] = "R"
+            else:
+                index = _mem.txtone - 1
+            mem.dtcs = DTCS[index]
+        else:
+            LOG.warn("Bug: txtone is %04x" % _mem.txtone)
+
+        if _mem.rxtone in [0, 0xFFFF]:
+            rxmode = ""
+        elif _mem.rxtone >= 0x0258:
+            rxmode = "Tone"
+            mem.ctone = int(_mem.rxtone) / 10.0
+        elif _mem.rxtone <= 0x0258:
+            rxmode = "DTCS"
+            if _mem.rxtone >= 0x6A:
+                index = _mem.rxtone - 0x6A
+                dtcs_pol[1] = "R"
+            else:
+                index = _mem.rxtone - 1
+            mem.rx_dtcs = DTCS[index]
+        else:
+            LOG.warn("Bug: rxtone is %04x" % _mem.rxtone)
+
+        if txmode == "Tone" and not rxmode:
+            mem.tmode = "Tone"
+        elif txmode == rxmode and txmode == "Tone" and mem.rtone == mem.ctone:
+            mem.tmode = "TSQL"
+        elif txmode == rxmode and txmode == "DTCS" and mem.dtcs == mem.rx_dtcs:
+            mem.tmode = "DTCS"
+        elif rxmode or txmode:
+            mem.tmode = "Cross"
+            mem.cross_mode = "%s->%s" % (txmode, rxmode)
+
+        mem.dtcs_polarity = "".join(dtcs_pol)
+
+        if not _mem.scan:
+            mem.skip = "S"
+
+        _levels = self.POWER_LEVELS
+        if _mem.txpower == TXPOWER_HIGH:
+            mem.power = _levels[0]
+        elif _mem.txpower == TXPOWER_MID:
+            mem.power = _levels[1]
+        elif _mem.txpower == TXPOWER_LOW:
+            mem.power = _levels[2]
+        else:
+            LOG.error('%s: get_mem: unhandled power level: 0x%02x' %
+                      (mem.name, _mem.txpower))
+
+        mem.mode = _mem.narrow and "NFM" or "FM"
+
+        mem.extra = RadioSettingGroup("Extra", "extra")
+
+        # BCL (Busy Channel Lockout)
+        rs = RadioSettingValueBoolean(_mem.bcl)
+        rset = RadioSetting("bcl", "BCL", rs)
+        mem.extra.append(rset)
+
+        # PTT-ID
+        rs = RadioSettingValueList(PTTID_LIST, PTTID_LIST[_mem.pttid])
+        rset = RadioSetting("pttid", "PTT ID", rs)
+        mem.extra.append(rset)
+
+        # Signal (DTMF Encoder Group #)
+        rs = RadioSettingValueList(PTTIDCODE_LIST, PTTIDCODE_LIST[_mem.scode])
+        rset = RadioSetting("scode", "PTT ID Code", rs)
+        mem.extra.append(rset)
+
+        # # Encrypt
+        # rs = RadioSettingValueList(ENCRYPT_LIST, ENCRYPT_LIST[_mem.encrypt])
+        # rset = RadioSetting("encrypt", "Encrypt", rs)
+        # mem.extra.append(rset)
```

Comment:
I really hate to have commented-out code like this when we can avoid it. Is this stuff you expect to enable later? It'd be nice to just keep that out in a separate commit in a branch that we can go back to later if/when you're ready. Reading the code without syntax highlighting on, it can be difficult to notice that this isn't actually live code, and since git can keep this sort of thing for us until later, we don't really need to "preserve" it here. Anyway, not too critical, but.. definitely my preference.

Reply:
Yeah... I meant to remove these before making the PR but doing it fell through the cracks.